doc/samplers.rst

```diff
@@ -30,6 +31,8 @@ The following table summarizes the (*d*-dimensional) sample spaces over which ea
     +--------------------------------------------+---------------------------------------+------------+--------------+------------+--------------+-----------------+
     | Method                                     | Sample Space                          | Univariate | Multivariate | Univariate | Multivariate | Transformations |
     +============================================+=======================================+============+==============+============+==============+=================+
+    | :ref:`ABC <section-ABC>`                   | :math:`\mathbb{R}^d`                  | No         | Yes          | No         | Yes          | Yes             |
+    +--------------------------------------------+---------------------------------------+------------+--------------+------------+--------------+-----------------+
```

Comment:
@bdeonovic - When you rebase your branch/PR on the latest documentation changes now in the master branch, note that the "Stand-Alone" and "Sampler-Based" columns of this table have been interchanged.  Also, since there is no stand-alone ABC sampler, both its Stand-Alone Univariate and Multivariate table entries should be "No".